packages/regl-worldview/src/Worldview.js

```diff
@@ -153,7 +157,13 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
     if (!this._tick) {
       this._tick = requestAnimationFrame(() => {
         this._tick = undefined;
+        if (this.props.onPaintStart) {
+          this.props.onPaintStart();
+        }
         worldviewContext.paint();
+        if (this.props.onPaintEnd) {
+          this.props.onPaintEnd();
```

Comment:
Should we consider computing the time taken and passing it to onPaintEnd for convenience?